Pages/Services.razor

```diff
@@ -37,7 +37,7 @@
 
 <hr>
 
-<div id="ketamineTherapy" class="row contentWrapper">
+<div id="Spravato" class="row contentWrapper">
```

Comment:
make css class lower camel case like the other css classes

Reply:
done, see new commit